<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSetting</class>
 <widget class="QDialog" name="DeviceSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多点远程超声 - 视频设备管理</string>
  </property>
  <widget class="QComboBox" name="deviceComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>341</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="deviceLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="capLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>格式/分辨率/帧率范围</string>
   </property>
  </widget>
  <widget class="QComboBox" name="capComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>179</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>179</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="colorspaceLabel">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>120</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>YUV Color Space</string>
   </property>
  </widget>
  <widget class="QComboBox" name="colorspaceComboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>deviceComboBox</tabstop>
  <tabstop>capComboBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
